lib/presto-client/index.js

```diff
@@ -42,7 +42,7 @@ var Client = exports.Client = function(args){
     if (args.custom_auth) {
         this.authorization = args.custom_auth;
     } else if (args.basic_auth) {
-        this.authorization = 'Basic ' + new Buffer(args.basic_auth.user + ':' + args.basic_auth.password).toString('base64');
+        this.authorization = 'Basic ' + new Buffer.from(args.basic_auth.user + ':' + args.basic_auth.password).toString('base64');
```

Comment:
`new Buffer.from` Is this correct?

Reply:
Yes @tagomoris Reference :- https://nodejs.org/api/buffer.html

Reply:
@aromalsanthosh According to the document, the call should be `Buffer.from` without `new` keyword.
```js
// Shares memory with `arr`.
const buf = Buffer.from(arr.buffer);
```
Is it really correct?

Reply:
@tagomoris  you're correct. The code has been updated accordingly.